resources/public/api/conf/1.0/common/errors.yaml

```diff
@@ -0,0 +1,94 @@
+components:
+  examples:
+
+    # 400
+
+    formatNino:
+      description: The format of the supplied NINO field is not valid.
+      value:
+        code: FORMAT_NINO
+        message: The provided NINO is invalid
+
+    formatTaxYear:
+      description: The format of the supplied taxYear field is not valid.
+      value:
+        code: FORMAT_TAX_YEAR
+        message: The provided tax year is invalid
+
+    formatSource:
+      description: The format of the supplied Source field is not valid.
+      value:
+        code: FORMAT_SOURCE
+        message: The provided source is invalid
+
+    formatValue:
+      description: One or more values have been added with the incorrect format.
+      value:
+        code: FORMAT_VALUE
+        message: The field should be between 0 and 99999999999.99
+
+    formatCustomerRef:
+      description: The format of customer reference is not valid.
+      value:
+        code: FORMAT_CUSTOMER_REFERENCE
+        message: The provided customer reference is not valid
+
+    ruleIncorrectOrEmptyBody:
+      description: An empty or non-matching body was submitted.
+      value:
+        code: RULE_INCORRECT_OR_EMPTY_BODY_SUBMITTED
+        message: An empty or non-matching body was submitted
+
+    ruleTaxYearNotSupported:
+      description: |
+        The specified tax year is not supported.
+        Either the tax year specified is before the minimum tax year value,
+        or it is after the maximum tax year value.
+      value:
+        code: RULE_TAX_YEAR_NOT_SUPPORTED
+        message: The tax year specified does not lie within the supported range
+
+    ruleTaxYearRangeInvalid:
+      description: Tax year range invalid. A tax year range of one year is required.
+      value:
+        code: RULE_TAX_YEAR_RANGE_INVALID
+        message: Tax year range invalid. A tax year range of one year is required
```

Comment:
The message should be `The tax year range is invalid` according to API #1668 spec

Reply:
Though this is correct according to spec API #1624

Reply:
errors.yaml is a common file, so can only be correct for one of them. However the spec should really be updated for the first one -- I'll add it to the checklist on the new confluence page